DOL.WHD.Section14c.Api/Web.config

```diff
@@ -52,6 +50,7 @@
   <system.webServer>
     <modules>
       <remove name="FormsAuthentication" />
+		  <add name="NLog" type="NLog.Web.NLogHttpModule, NLog.Web" />
```

Comment:
@mgwalker I don't see the extra spacing from my Visual Studio.

Reply:
It's spaces vs. tab.  Github renders tabs as 8 spaces.  If your VS only renders them as 4, that's why you wouldn't see it.  The prior lines in `Web.config` use spaces instead of tabs.  Either way is fine with me, as long as it's consistent.

Reply:
The extra spaces have been removed.

Reply:
@dciceman If not already JIC, we can also request VS to not keep tabs but spaces in tools->options ->...-> Text Editor

Reply:
I see the Nlog section is lined up correctly from the link below. Am I looking at the wrong place?

https://github.com/18F/dol-whd-14c/pull/316/files/84585f75652812b95532e0bfeec0025d0c7978ec#diff-6579a214eace0b3028112e6897353bcb